department-of-veterans-affairs/caseflow
# Add Caseflow Search to Reader

Resolves #8926 

### Description
Added a link to search cases to the top level navigation bar in Reader

### Acceptance Criteria
- [X] Caseflow search link appears in the navigation bar for Reader. See screenshot below for clarification.

![screen shot 2019-02-12 at 10 33 36 am](https://user-images.githubusercontent.com/45575454/52647239-56649700-2eb2-11e9-94ce-a9fa443592c9.png)

### Testing Plan
1. Open any appeal's document list and confirm the "Search Cases" link appears in the top right corner of the the navigation bar.
2. Open any document from the document list and confirm the "Search Cases" link appears in the top right corner of the the navigation bar.
